src/components/LandingFilter.jsx

```diff
@@ -0,0 +1,128 @@
+/* eslint-env jquery */
+import React, { useState, useEffect } from 'react';
+import { FiArrowRightCircle } from 'react-icons/fi';
+import {
+  Dropdown, Grid, Form
+} from 'semantic-ui-react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import theme from '../config/theme';
+import { regionOptions, paxOptions, ratingOptions } from '../config/constant';
+
+const GoButton = styled(Link)`
+  font-size: 50px;
+  font-weight: bold;
+  display: inline-flex;
+  align-items: center;
+  color: ${theme.colours.maroon};
+  :hover {
+    color: ${theme.colours.maroon};
+    opacity: 0.8;
+  }
+`;
+
+const LandingFilter = ({ category } = {}) => {
+  const [values, setValues] = useState({
+    keyword: '',
+    region: '',
+    pax: '',
+    rating: '',
+    price: '100'
+  });
+  const onChange = (e, result) => {
+    const { name, value } = result || e.target;
+    setValues({ ...values, [name]: value });
+  };
+
+  useEffect(() => {
+    $('#amount-slider').range({
```

Comment:
Is it possible to not mix raw css with react? Perhaps there is a slider plugin we can import?